src/Codeception/Module/REST.php

```diff
@@ -861,6 +883,24 @@ public function seeResponseIsValidOnJsonSchema($schema)
         );
     }
 
+    /**
+     * Checks whether last response matches the supplied json schema (https://json-schema.org/)
+     * Supply schema as filepath in your codeception data directory.
```

Comment:
If the project is truly using JsonSchema for validating its inputs, schema files won't be in tests directory and copying them there causes unnecessary duplication.
I looked at 3 most popular dependents of justinrainbow/json-schema library:
https://github.com/composer/composer/tree/master/res
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/doc/schema.json
https://github.com/drupal/core/blob/9.0.x/modules/jsonapi/schema.json

I think that it would be the best to use [codecept_absolute_path](https://github.com/Codeception/Codeception/blob/4.0.3/autoload.php#L93-L104) function and replace the line above with
`Supply schema as relative file path in your project directory or an absolute path`

Reply:
Not sure if it will always be in the project directory. When generating JsonSchema responses you probably don't validate during generation. But it's good to have that option.  Changes are pushed.